### DISCUSSION TITLE
Better mustache/jinja support in langfuse playground

### DISCUSSION THREAD

**Discussion by ghost** [langfuse/langfuse#3396] [Ideas]
### Describe the feature or potential improvement

I noticed support for mustache variables detection in the langfuse UI is currently quite limited.
E.g. For a conditional variable statement like this
```mustache
{{#some_variable}}It exists: {{.}}{{/some_variable}}
```
The variable `some_variable` is not listed in the UI and not usable in the playground.

It would be nice if full mustache syntax is properly supported everywhere.

**Comment by marcklingen** [comment]
I agree that this would be cool! I saw other teams using Jinja to incorporate loops and conditional logic in their prompts.

**Reply by marcklingen** [reply]
I updated the title to include jinja as well

**Comment by DannyGsGit** [comment]
+1 vote, my team has been using a lot of conditional prompts with Jinja templates. We have a hacky way of using these templates through Langfuse today (our own prompt compile), but it would be amazing if we could set Jinja templating engine in compile().

**Comment by atritch** [comment]
@marcklingen, this appears to be one of the only significant features your platform lacks for our use case. Is there any chance this has made it on your roadmap since this was requested?

**Comment by fcalabrow** [comment]
this feature would be awesome!

**Comment by recursingfeynman** [comment]
+1 for jinja

**Comment by kimdebie** [comment]
Same here!
